bookkeeper-server/src/main/java/org/apache/bookkeeper/client/EnsemblePlacementPolicy.java

```diff
@@ -423,7 +423,9 @@ default boolean isEnsembleAdheringToPlacementPolicy(List<BookieSocketAddress> en
      *            list of BookieSocketAddress of bookies that have acknowledged a write.
```

Comment:
nit: you forgot to mention added params to the comment section

Reply:
Fixed